src/main/java/com/firebase/geofire/AndroidEventRaiser.java

```diff
@@ -0,0 +1,17 @@
+package com.firebase.geofire;
+
+import android.os.Handler;
+import android.os.Looper;
```

Comment:
These imports would fail on non android platforms, i'm not that familiar with java, is there a way to exclude this and fallback on ThreadEventRaiser with some compile time conditional?